packages/config/src/config.ts

```diff
@@ -21,9 +21,11 @@ const tokenPropertyRegex = /(?<!var\()--((\w)([\w-]+)?)/;
 const variantPropertyRegex = /(?<!var\()--((\w)([\w-]+)_([\w-]+))/;
 const tokenValueRegex = /var\(--([\w-]+)_([\w-]+)\)/;
 const aritraryValueRegex = /var\(---,(.+)\)/;
+const gridValueRegex = /^\d+/;
 
-type GridValue = v.Output<typeof gridValueSchema>;
-const gridValueSchema = v.number();
+type GridValue = number;
+const gridValueRegexSchema = v.regex(gridValueRegex);
+const gridValueSchema = v.string([gridValueRegexSchema]);
```

Comment:
i need to transform this to `number` at some stage but i didn't have time to figure out the valibot api for that. will revisit later if/when this becomes a problem.